en/support/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Version Support
+menu: support
+lang: en
+---
+
+# Version Support
+
+Only the latest version of any given major release line is supported.
+
+Versions that are EOL (end of life) _may_ receive updates for critical security vulnerabilities, but the Express team offers no guarantee and has no intention of fixing any issues found or releasing any fixed versions.
```

Comment:
```suggestion
Versions that are EOL (end-of-life) _may_ receive updates for critical security vulnerabilities, but the Express team offers no guarantee of fixing any issues found or releasing any fixed versions.
```

Reply:
Typically “EOL’ indeed means security fixes are intended to go unfixed.

Reply:
Yeah I think I prefer the explicit and direct "we do not intend to patch security fixes in EOL versions".

Reply:
I have every intention to patch critical security issues

Reply:
Is that the project's position?

Either way, I would think you'd want to message "no intention" regardless of your actual intention, to encourage upgrades.

Reply:
updated suggestion as a compromise

Reply:
I think it is better to under promise and over deliver, but I do agree it would be awesome if we have folks around to patch critical things as best we can.

Reply:
BTW, I am good with this language. My comment was not exactly clear there. 👍 landing this suggestion.

Reply:
ok, suggestion pulled in. it's semantically identical but a bit connotationally weaker, which i think is a downside, but nbd.